### PR TITLE
Add EntityRedirect::__toString for easy debugging

### PR DESCRIPTION
**Pull request by thiemowmde**

### CODE REVIEW

**Comment by Benestar** [comment]
Do we have __toString methods anywhere else? I'd suggest to first define generally if and how we want __soString to be defined in our datamodel and then implement it for all models we think it makes sense?

**Comment by thiemowmde** [comment]
We have it in many cases where it makes sense, e.g. Entity::__toString. It probably doesn't make much sense for more complex classes like Fingerprint.

**Comment by JeroenDeDauw** [comment]
@thiemowmde did you mean EntityId or something in your last comment? Else I don't get the remark on complexity, as Entity is more complex than Fingerprint

File: src/Entity/EntityRedirect.php [hunk 1]
```diff
@@ -71,4 +71,8 @@ public function equals( $that ) {
 			&& $this->targetId->equals( $that->targetId );
 	}
 
+	function __toString() {
```
Comment (JeroenDeDauw) [inline src/Entity/EntityRedirect.php]: public function

**Comment by JeroenDeDauw** [comment]
Needs test to detect regressions and be documented as a new thing in the packages public interface

**Pull request merged**